### DISCUSSION TITLE
O livro Coletânea Front-end vale a pena?

### DISCUSSION THREAD

**Discussion by thiamsantos** [frontendbr/forum#263] [Discussão]
Eu estou pensando em comprar o livro [Coletânea Front-end](https://www.casadocodigo.com.br/products/livro-coletanea-front-end), eu estou ciente da qualidade dos livros da casa do código, mas gostaria de saber da opinião pessoal de alguém. 

Alguém já leu?
Vale a pena?
Posso comprar sem medo?

**Comment by afonsopacifer** [comment]
@thiamsantos SIM SIM SIM!

Kra, esse livro tem tanto conteúdo bom, que eu nem sei o que dizer, apenas compre, leia, e seja feliz ;)

![photo on 7-26-16 at 8 19 pm](https://cloud.githubusercontent.com/assets/6748866/17158568/9b8ee7d6-536d-11e6-8b46-91ee43836649.jpg)

**Comment by hevertoncastro** [comment]
Ele é ótimo para usar como guia de estudo. Há vários tópicos de assuntos relevantes e atuais onde os autores fazem um overview de cada. O interessante é saber do que se trata e continuar sua pesquisa com essa base.

**Comment by hevertoncastro** [comment]
Recomendo também:
- [Guia Front-End - O caminho das pedras para ser um dev Front-End](https://www.casadocodigo.com.br/products/livro-guia-frontend)
- [CSS Eficiente Técnicas e ferramentas que fazem a diferença nos seus estilos](https://www.casadocodigo.com.br/products/livro-css-eficiente)

**Comment by thiamsantos** [comment]
@afonsopacifer @hevertoncastro muito obrigado pelos comentários !!!

Acabei de encomendar o livro :smile:

**Comment by leandroruel** [comment]
é so overview mesmo como  o amiguinho ali disse
